freqtrade/exchange/__init__.py

```diff
@@ -571,6 +566,13 @@ def refresh_latest_ohlcv(self, pair_list: List[Tuple[str, str]]) -> List[Tuple[s
                 ticks, tick_interval, fill_missing=True)
         return tickers
 
+    def _now_is_time_to_refresh(self, pair: str, ticker_interval: str) -> bool:
```

Comment:
Honestly - GREAT NAME!

Reply:
Thoroughly chosen... :-)

I named it _time_to_refresh() at first try but decided that a function with such the name should return *time*, while it returns bool...